### PR TITLE
[HDInsight] Hotfix for deprecating Dv1 series: change Dv1-series to related Dv2

### PR DESCRIPTION
**Pull request by aim-for-better**
change D3 to D3_v2, D12 to D12_v2.
there is no related swagger spec pr.

### CODE REVIEW

File: sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Microsoft.Azure.Management.HDInsight.csproj [hunk 1]
```diff
@@ -7,14 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>5.1.0</Version>
+		<Version>5.2.0</Version>
```
Comment (isra-fel) [inline sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Microsoft.Azure.Management.HDInsight.csproj]: For hotfix the patch (third) version should be updated, i.e. `5.1.1`

File: sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@
 [assembly: AssemblyTitle("Microsoft Azure HDInsight Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure HDInsight.")]
 [assembly: AssemblyVersion("5.0.0.0")]
-[assembly: AssemblyFileVersion("5.1.0.0")]
+[assembly: AssemblyFileVersion("5.2.0.0")]
```
Comment (isra-fel) [inline sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Properties/AssemblyInfo.cs]: here too. thanks!

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Microsoft.Azure.Management.HDInsight.csproj (hunk 1 above)
Comment (aim-for-better) [reply sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Microsoft.Azure.Management.HDInsight.csproj]: > For hotfix the patch (third) version should be updated, i.e. `5.1.1`

Fixed. Thanks

File: sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (aim-for-better) [reply sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Properties/AssemblyInfo.cs]: > here too. thanks!

Fixed.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**